integration-test/src/main/resources/testsuites/e2e/ssh-tests.yaml

```diff
@@ -0,0 +1,5 @@
+name: "ssh-tests"
+tests:
+  - name: "ssh_tests"
+    classes:
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.ssh.SshSudoCommandTest
```

Comment:
do we need a separated test suite? probably you will add the new test to an existing test suite

Reply:
We discussed it with @aszegedi, we can't see any concern on having this as a separated test suite. For sure, if you have a specific one what you think we should extend, please share the suite name an I can move the testcase there.

Reply:
if you will have separated jenkins job for this, separated suite should be the way. if you will use existing job, you should use the related suite.
but anyway, this is just a note, as the owner of the feature, you can choose the most sympathetic solution for you